.asf.yaml

```diff
@@ -33,4 +33,5 @@ notifications:
     issues:       issues@iceberg.apache.org
     pullrequests: issues@iceberg.apache.org
 
-
+publish:
+    whoami:  asf-site
```

Comment:
Nit: a newline so we don’t get the red mark? I’m not sure how much people care but I don’t think see them in the main repo.